tests/strainMeasurement/makeAll.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+acciv-makeGeometryFactors images/image000.h5 images/gridGeometryFactors.h5 flat
```

Comment:
The names `acciv-makeGeometryFactors` and `acciv` are specific to your setup.  I would suggest changing them to give the relative path the `makeGeometryFactors.exe` and `acciv.exe` as they build by default.